src/ducks/settings/GroupSettings.jsx

```diff
@@ -49,18 +52,20 @@ class GroupSettings extends Component {
     }
   }
 
-  toggleAccount = (accountId, enabled) => {
-    const { group, getAssociation } = this.props
-    const accounts = getAssociation(group, 'accounts')
+  toggleAccount = async (accountId, group, enabled) => {
+    const accounts = group.accounts
     if (enabled) {
       accounts.addById(accountId)
     } else {
       accounts.removeById(accountId)
     }
+    if (!group.id) {
+      this.updateOrCreate(dehydrateDoc(group))
+      this.setState({})
```

Comment:
What's the purpose of this `setState` ?

Reply:
It should not be necessary after the work I did on the queries, I am going to try thanks !